src/config.h.in

```diff
@@ -0,0 +1,3 @@
+#if XAPIAN_MAJOR_VERSION >= 2 || (XAPIAN_MINOR_VERSION >= 3 && XAPIAN_REVISION >= 6)
```

Comment:
I think you want `|| XAPIAN_MINOR_VERSION >=3 || (XAPIAN_MINOR_VERSION == 3 && XAPIAN_REVISION >= 6)`.

(Otherwise `1.5.0` will fail this test.)